src/main/interface/msp.c

```diff
@@ -1996,10 +1996,10 @@ mspResult_e mspProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
         gyroConfigMutable()->imuf_yaw_q = sbufReadU16(src);
         gyroConfigMutable()->imuf_w = sbufReadU16(src);
 #ifdef USE_GYRO_IMUF9001
-        gyroConfigMutable()->imuf_roll_lpf_cutoff_hz = sbufReadU16(src);
-        gyroConfigMutable()->imuf_pitch_lpf_cutoff_hz = sbufReadU16(src);
-        gyroConfigMutable()->imuf_yaw_lpf_cutoff_hz = sbufReadU16(src);
-        gyroConfigMutable()->imuf_acc_lpf_cutoff_hz = sbufReadU16(src);
+        sbufReadU16(src);
```

Comment:
same here?

Reply:
basically with how the MSP works if you remove something you need to keep a "blank spot" for it to just read past otherwise it will get out of sync.

Reply:
ok i see. cant this be optimized somehow, like having one expression instead of three each?

Reply:
I don't think it easily can, refer to this part of the current msp.c
https://github.com/emuflight/EmuFlight/blob/556f8c6ce906e2a469142d629434203fcebf8bef/src/main/interface/msp.c#L778

Reply:
https://github.com/emuflight/EmuFlight/blob/556f8c6ce906e2a469142d629434203fcebf8bef/src/main/interface/msp.c#L1155